docs/operation.md

```diff
@@ -0,0 +1,1100 @@
+| [searchCondition](#search-condition-of-a-patient)  | Search Medical condition of a patient.|
+| [searchContract](#search-contract)  | Search contracted people who are authorized to view patient details.|
+| [SearchDevice](#search-implanted-device-details)  | Search implanted device history of a patient.|
+| [searchDiagnosticReport](#search-diagnostic-reports-of-a-patient)  | Search information on diagnostic report and procedures.|
+| [searchDocumentReference](#search-document-reference)  | Search clinical documents and reference to retrieve them.|
+| [searchGoal](#search-goal)  | Search intended objectives for a patient.|
+| [searchImmunization](#search-immunization-for-a-patient)  | Search current and historical administration of vaccinations.|
+| [searchMedicationAdministration](#search-medication-administration-of-a-patient)  | Search Medications and vaccines administrated to a patient.|
+| [searchMedicationOrder](#search-medication-order)  | Search orders for all medications along with administration instructions.|
+| [searchMedicationStatement](#search-medication-statement)  | Search medications taken by the patient now or in the past.|
+| [searchNewEncounter](#search-encounter)  | Search admissions or visits during which health care services were provided to a patient.|
+| [searchObservation](#search-observation)  |Search measurements or simple assertions about a patient.|
+| [searchPatient](#search-patient)  | Search about a person receiving health care services from a specific organization.|
+| [searchPractitioner](#search-practitioner)  | Search information about a person formally involved in the care of a patient.|
+| [searchProcedure](#search-procedure)  | Search current and historical procedures performed on a patient.|
+| [searchRelatedPerson](#search-related-person-of-a-patient)  | Search information about a person who is involved in the care of a patient but has no formal responsibility|
+| [searchSchedule](#search-schedule)  | Search  time slots which are defined for booking an appointment.|
+| [searchSlot](#search-slot)  | Search time slots from a schedule which are available for booking an appointment.|
+
+
+
+
+## Operation details
+This section provides details on each of the operations.
+
+## Retrieve a specific resource by its Id
+This retrieveById operation reads a resource based on the Id you provide.
+
+**retrieveById**
+```xml
+<cerner.retrieveById>
+    <base>{$ctx:base}</base>
+    <type>{$ctx:type}</type>
+    <id>{$ctx:id}</id>
+</cerner.retrieveById>
+```
+**Properties**
+* base: The service root URL.
+* type: The type of the resource that is needed to be retrieved.
+* Id: The logical Id of the resource.
+
+**Sample request**
+
+Following is a sample REST request that can be handled by the retrieveById operation.
+```
+{
+ "base": “https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/",
+ "type": “Patient",
+ "id": "4342008"
+}
+```
```

Comment:
Add sample response for each operation as in [1].
[1] https://docs.wso2.com/display/ESBCONNECTORS/Working+with+Sheet+Operations